vendor/BUILD

```diff
@@ -43,7 +43,6 @@ filegroup(
         "//vendor/github.com/matttproud/golang_protobuf_extensions/pbutil:all-srcs",
         "//vendor/github.com/peterbourgon/diskv:all-srcs",
         "//vendor/github.com/prometheus/client_golang/prometheus:all-srcs",
-        "//vendor/github.com/prometheus/client_model:all-srcs",
```

Comment:
Shouldn't this also update the dep files?

Reply:
https://github.com/kubernetes/test-infra/blob/cab53df75a87c008fafb0a6036652110905656c4/Gopkg.lock#L187

Reply:
yeah, it should have.

Reply:
Also, this library seems to be used transitively by `github.com/prometheus/client_golang`.

Reply:
OK, this seems to be added back in this very same PR. Sorry for the noise.